dev-java/jansi/jansi-2.4.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml --download-uri https://github.com/fusesource/jansi/archive/refs/tags/jansi-2.4.0.tar.gz --slot 0 --keywords "~amd64 ~arm ~arm64 ~ppc64 ~x86" --ebuild jansi-2.4.0.ebuild
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="org.fusesource.jansi:jansi:2.4.0"
+
+inherit java-pkg-2 java-pkg-simple toolchain-funcs
+
+DESCRIPTION="Jansi is a java library for generating and interpreting ANSI escape sequences."
+HOMEPAGE="http://fusesource.github.io/jansi"
+SRC_URI="https://github.com/fusesource/${PN}/archive/refs/tags/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="2"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+# Compile dependencies
+# POM: pom.xml
+# test? info.picocli:picocli-codegen:4.5.2 -> !!!artifactId-not-found!!!
+# test? org.junit.jupiter:junit-jupiter:5.7.0 -> !!!groupId-not-found!!!
+# test? org.junit.jupiter:junit-jupiter-params:5.7.0 -> !!!groupId-not-found!!!
+
+DEPEND=">=virtual/jdk-1.8:*"
+
+# junit-jupiter is not available in ::gentoo
+#	test? (
+#		!!!artifactId-not-found!!!
+#		!!!groupId-not-found!!!
+#	)
+#"
+
+RDEPEND=">=virtual/jre-1.8:*"
+
+DOCS=( {changelog,readme}.md license.txt )
+
+S="${WORKDIR}/${PN}-${P}"
+
+JAVA_SRC_DIR="src/main/java"
+JAVA_RESOURCE_DIRS="src/main/resources"
+JAVA_MAIN_CLASS="org.fusesource.jansi.AnsiMain"
+
+# junit-jupiter is not available in ::gentoo
+#JAVA_TEST_GENTOO_CLASSPATH="!!!artifactId-not-found!!!,!!!groupId-not-found!!!,!!!groupId-not-found!!!"
+#JAVA_TEST_SRC_DIR="src/test/java"
+#JAVA_TEST_RESOURCE_DIRS="src/test/resources"
+
+src_prepare() {
+	default
+	rm -r "${JAVA_RESOURCE_DIRS}/org/fusesource/jansi/internal/native" || die
```

Comment:
could you please comment why we need to remove this?

Reply:
Done https://github.com/gentoo/gentoo/compare/d38e0fde4bd3157d380e225b33ea60b745ff8688..078d9cc32c7bbb94c0349066b773ccb511fb58a8#diff-1ce1c8af1dc228f0228e8b8a5b31bc719c5cef45e6cd5658633f8c6119f350b3